profiler/include/profiler/profile_gemm_universal_impl.hpp

```diff
@@ -90,9 +91,16 @@ bool profile_gemm_universal_impl(int do_verification,
     const auto b_element_op = BElementOp{};
     const auto c_element_op = CElementOp{};
 
-    DeviceMem a_device_buf(sizeof(ADataType) * a_m_k.mDesc.GetElementSpaceSize());
-    DeviceMem b_device_buf(sizeof(BDataType) * b_k_n.mDesc.GetElementSpaceSize());
-    DeviceMem c_device_buf(sizeof(CDataType) * c_m_n_device_result.mDesc.GetElementSpaceSize());
+    int total_gemm_needed = a_m_k.GetElementSpaceSizeInBytes() +
+                            b_k_n.GetElementSpaceSizeInBytes() +
+                            c_m_n_device_result.GetElementSpaceSizeInBytes();
+    int rotating_count = std::max(static_cast<double>(1.),
+                                  std::ceil(static_cast<double>(rotating) / total_gemm_needed));
```

Comment:
rotating_count should be smaller or equal than niter.

Reply:
done